CMakeModules/ODInternal.cmake

```diff
@@ -236,3 +236,47 @@ if ( NOT ${BUILDINSRC} )
 			    PATTERN "*.ico" PATTERN "*.rc" PATTERN "*.txt"
 			    EXCLUDE PATTERN CMakeFiles EXCLUDE )
 endif()
+
+if ( APPLE )
+
+	set(LIBRARY_DESTINATION "${CMAKE_BINARY_DIR}/Contents/Frameworks/debug")
+	file(MAKE_DIRECTORY ${LIBRARY_DESTINATION})
+
+	if (TARGET Freetype::Freetype)
```

Comment:
This should probably best be a cmake function, taking the target name and module name as input:
copy_target_libs( target modulename )

The macro would defined a post build custom command of type TARGET, for ${OD_MODULE_NAME}, to be called inside the add_fontconfig function (ODUtils.cmake:498)